tests/cycpp_tests.py

```diff
@@ -390,74 +393,162 @@ def test_itdbfilter():
     yield assert_equal, exp_args, args
 
     impl = f.impl()
-    exp_impl = ('  int rawcycpp_shape_y[1] = {42};\n'
-                '  cycpp_shape_y = std::vector<int>(rawcycpp_shape_y, '
-                                                   'rawcycpp_shape_y + 1);\n'
-                '  tree = tree->SubTree("config/*");\n'
-                '  cyclus::InfileTree* sub;\n'
-                '  int i;\n'
-                '  int n;\n'
-                '  x = cyclus::Query<int>(tree, "x");\n'
-                'THINGFISH\n'
-                '  di.NewDatum("Info")\n'
-                '  ->AddVal("x", x)\n'
-                'ABSOLUTELY FREE\n'
-                '  ->Record();\n')
-    yield assert_equal, exp_impl, impl
+    exp_impl = (
+        '  int rawcycpp_shape_y[1] = {42};\n  cycpp_shape_y = std::vector<int>(rawcycpp_shape_y, rawcycpp_shape_y + 1);\n  cyclus::InfileTree* sub = tree->SubTree("config/*");\n  int i;\n  int n;\n  {\n    int x_val = cyclus::Query<int>(sub, "x");\n    x = x_val;\n  }\nTHINGFISH\n  di.NewDatum("Info")\n  ->AddVal("x", x)\nABSOLUTELY FREE\n  ->Record();\n'
```

Comment:
This statement is too long for one line and it is on three line.  You may as well split up the string since you already have the grouping parentheses.